lab1.js

```diff
@@ -52,6 +52,10 @@ assert(1 === 2, "this is an assertion failure example. 1===2");
  it failed.
 */
 
+// Below is my code
+assert("Woodland Park Zoo" === "Woodland Park Zoo");
+assert("Woodland Park Zoo" === "woodland park zoo", "JS is case-sensitive therefore it failed.");
```

Comment:
Glad you did an example with case-sensitivity. That's a nuance of the language that will bite you if you're not paying attention. :)